### DISCUSSION TITLE
Eloquent driver only adds Entries to database

### DISCUSSION THREAD

**Discussion by sander-vth** [statamic/cms#6820] [Troubleshooting] [answered]
HI all,

I would like to have my fresh installation of Statamic to have all items stored in MySQL instead of files.
So I installed the statamic/eloquent-driver and walked through the installation steps.

After it was finished it had only made a migration file for Entries (not Collections, forms, etc.).
Also in the config only Entries was specified.

What can I do to make it have all items stored in a database?

**Comment by jasonvarga** [comment]
Please open an issue on the eloquent driver repo.

https://github.com/statamic/eloquent-driver

**Comment by ryanmitchell** [comment]
@webigniters are you definitely using v1 or higher of eloquent driver? It sounds to me like you might be stuck on v0.2

**Comment by sander-vth** [comment, marked as answer]
I have opened an issue on the driver repo.
I don't know which version I use. I just followed the installation steps mentioned on GitHub.
https://github.com/statamic/eloquent-driver

**Reply by duncanmcclean** [reply]
Looks like you managed to get this sorted on the Eloquent Driver repository: https://github.com/statamic/eloquent-driver/issues/62.

I'll mark this comment as the answer!
